### PR TITLE
Time picker

### PR DESCRIPTION
**Pull request by Emmapr123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +24,10 @@ npm install @react-navigation/stack
 
 npm install axios
 
+## Time Picker
+
+npm install react-time-picker
+
```
Comment (TarandeepNandhra) [inline README.md:36]: Would be helpful to have the rough steps to make a new component or screen

File: src/SVGs/AddNewWorkoutButton.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      />
+      <Path
+        d="M33.53 19.67l-2.08 6.28h-2.68l6.82-20h3.12l6.84 20h-2.76l-2.14-6.31zm6.58-2l-2-5.77c-.45-1.31-.75-2.5-1-3.66h-.05c-.3 1.19-.63 2.41-1 3.63l-2 5.8zM48.5 6.19a36.88 36.88 0 015.5-.41c3.72 0 6.37.86 8.13 2.5a9.23 9.23 0 012.82 7.2 10.6 10.6 0 01-2.88 7.79c-1.88 1.88-5 2.89-8.87 2.89a41.49 41.49 0 01-4.7-.24zm2.59 17.76a16.29 16.29 0 002.62.14c5.53 0 8.54-3.09 8.54-8.5 0-4.74-2.65-7.74-8.13-7.74a14.5 14.5 0 00-3 .27zM68.34 6.19a37 37 0 015.51-.41c3.72 0 6.37.86 8.12 2.5a9.2 9.2 0 012.83 7.2 10.57 10.57 0 01-2.89 7.79c-1.87 1.88-5 2.89-8.87 2.89a41.49 41.49 0 01-4.7-.21zm2.56 17.76a16.43 16.43 0 002.62.14c5.54 0 8.54-3.09 8.54-8.5 0-4.74-2.65-7.74-8.12-7.74a14.68 14.68 0 00-3 .27z"
+        fill={color}
+      />
+    </Svg>
+  )
+}
+
+export { AddNewWorkoutButton }
```
Comment (TarandeepNandhra) [inline src/SVGs/AddNewWorkoutButton.js:33]: Nice component, would be helpful to have some comments to explain what each bit does

File: src/screens/AddNewWorkoutScreen/AddNewWorkoutScreen.js [hunk 3]
```diff
@@ -1,26 +1,108 @@
...
-import React from 'react';
-import { Button, SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import {Picker} from '@react-native-picker/picker';
+import { Button, KeyboardAvoidingView, SafeAreaView, ScrollView, StyleSheet, Text, TextInput, View } from 'react-native';
 
 const AddNewWorkoutScreen = () => {
 
+  const [workoutTitle, setWorkoutTitle] = useState('')
+  const [excersizeTitle, setExcersizeTitle] = useState('')
```
Comment (TarandeepNandhra) [inline src/screens/AddNewWorkoutScreen/AddNewWorkoutScreen.js]: would more consistent to use British English spelling for when making the post request

File: src/screens/AddNewWorkoutScreen/AddNewWorkoutScreen.js [hunk 4]
```diff
@@ -1,26 +1,108 @@
...
+        style={{backgroundColor: 'white', flex: 1}}
+        selectedValue={seconds}
+        onValueChange={(itemValue, itemIndex) =>
+          setSeconds(itemValue)
+        }>
+        <Picker.Item label="45" value={30} />
+        <Picker.Item label="30" value={45} />
+        <Picker.Item label="15" value={15} />
+        <Picker.Item label="00" value={0} />
+      </Picker>
```
Comment (TarandeepNandhra) [inline src/screens/AddNewWorkoutScreen/AddNewWorkoutScreen.js:53]: Super cool!!!

File: src/screens/WorkoutListScreen/WorkoutListScreen.js [hunk 5]
```diff
@@ -15,27 +27,30 @@ const WorkoutListScreen = ( {navigation} ) => {
...
-      onPress={ () => navigation.navigate('Add New Workout')}
-      />
+    <View style={{flex: 1, position: 'relative'}}>
+    <ScrollView>
+      <WorkoutList workouts={workouts} {...{getWorkouts}} />
     </ScrollView>
+    <TouchableOpacity onPress={() => {setPressed(true); navigation.navigate('Add New Workout');}}>
+      <AddNewWorkoutButton style={styles.button} color={pressed == true ? 'red' : 'blue'}/>
+    </TouchableOpacity>
+    </View>
```
Comment (TarandeepNandhra) [inline src/screens/WorkoutListScreen/WorkoutListScreen.js:37]: Cool logic

**Review by TarandeepNandhra — COMMENTED** [review]
Awesome stuff, would be helpful to have the rough steps in a README so it's easier to follow the PR

File: src/SVGs/AddNewWorkoutButton.js (hunk 2 above)
Comment (ralphbartley) [reply src/SVGs/AddNewWorkoutButton.js:33]: Cool will add comment about origin of SVG data

File: README.md (hunk 1 above)
Comment (ralphbartley) [reply README.md:36]: will link to tutorial

File: src/screens/AddNewWorkoutScreen/AddNewWorkoutScreen.js (hunk 3 above)
Comment (ralphbartley) [reply src/screens/AddNewWorkoutScreen/AddNewWorkoutScreen.js]: mmmhmm we'll fix the lyzdexia

**Pull request merged**